docs/user-guide.md

```diff
@@ -410,6 +541,16 @@ TODO
 
 ## Label propagation
```

Comment:
Label propagation algorithm (LPA)

It is useful to provide the wikipedia link.